util/chplenv/utils.py

```diff
@@ -23,6 +23,13 @@ def get_chpl_home():
     return chpl_home
 
 @memoize
+def using_chapel_module():
```

Comment:
Not your fault since the original code is mine, but I wonder if this would be cleaner as: 

```python
def using_chapel_module():
    chpl_home = os.environ.get('CHPL_HOME')
    if chpl_home != None:
        return chpl_home == os.environ.get('CHPL_MODULE_HOME')
    return False
```